server/util/flagutil/flagutil.go

```diff
@@ -34,3 +36,9 @@ func ResetFlags() error {
 func GetDereferencedValue[T any](name string) (T, error) {
 	return common.GetDereferencedValue[T](common.DefaultFlagSet, name)
 }
+
+// Expand updates the flag value to replace any placeholders in format ${FOO}
+// with the content of calling the mapper function with the placeholder name.
+func Expand(v flag.Value, mapper func(string) (string, error)) error {
+	return common.Expand(v, mapper)
+}
```

Comment:
nit: could we call `common.Expand` directly instead of using this func?

Reply:
The indirect function is not required, and I've changed this to an assignment, but `common` is intentionally not accessible outside of `flagutil` and its subpackages, so we do need to basically "export" it here.